install/installer/pkg/components/server/configmap.go

```diff
@@ -64,6 +64,14 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil
 	})
 
+	runDbDeleter := true
+	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
+		if cfg.WebApp != nil && cfg.WebApp.Server != nil {
+			runDbDeleter = cfg.WebApp.Server.RunDbDeleter
```

Comment:
IMO wee need to check for nil, and either:
 - default to true (again)
 - or (better) add the != nil check to the if check.

Reply:
Not sure I follow 😕 . `RunDbDeleter` is a `bool` field so can't be `nil`.

Reply:
Woops, my bad.
What I meant is we should distinguish between false (default value in case not specified) and actively set to false. The former we need to identify and in ignore.
Maybe by making that field a `*bool`?

Reply:
Rebased to make it a pointer. The same also applies to `enableLocalApp` which should also default to `true` when unset (8d1af9f7c8e74645f40cf44e3daeb6969dd2c392).